packages/scene-composer/src/augmentations/components/three-fiber/anchor/__tests__/__snapshots__/AnchorWidget.spec.tsx.snap

```diff
@@ -13,7 +13,9 @@ exports[`AnchorWidget should render correctly 1`] = `
       }
     }
   >
-    <lineSegments>
```

Comment:
I have added a test to mock onWidgetClick by mocking the event. Maybe we can add similarly for line segment too? 
https://github.com/awslabs/iot-app-kit/blob/main/packages/scene-composer/src/augmentations/components/three-fiber/anchor/__tests__/AnchorWidget.spec.tsx#L241

Reply:
Yep, let me add that since we already have it set up.